### PR TITLE
[Promotions] - move promotions to the its own component

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.monday.com/boards/2218344956/pulses/2711744595?doc_id=2711756418&userId=26728963

### CODE REVIEW

**Review by creed-victor — APPROVED** [review]

**Comment by tiltom** [comment]
I think that there is probably too much spacing on yield farm page.
image

File: src/app/components/Promotions/index.tsx [hunk 1]
```diff
@@ -22,59 +22,60 @@ export const Promotions: React.FC = () => {
...
             appSection={AppSection.YieldFarm}
             backgroundColor={PromotionColor.Orange}
-            title="5K SOV rewards"
-            duration="Ongoing weekly rewards"
-            text="Provide a 1:1 ratio of MYNT and rBTC to the MYNT/BTC AMM liquidity pool and instantly start accruing your share of 5,000 SOV rewards."
-            learnMoreLink="https://www.sovryn.app/blog/sovryn-mynt-project-updates"
+            title={t(translations.promotions.card1.title)}
+            duration={t(translations.promotions.card1.duration)}
+            text={t(translations.promotions.card1.text)}
+            learnMoreLink="https://wiki.sovryn.app/en/sovryn-dapp/market-making#yield-farming"
```
Comment (tiltom) [inline src/app/components/Promotions/index.tsx:28]: I would extract the link to `utils/classifiers`.

File: src/app/components/Promotions/index.tsx [hunk 2]
```diff
@@ -22,59 +22,60 @@ export const Promotions: React.FC = () => {
...
             backgroundColor={PromotionColor.Yellow}
-            title="15K SOV rewards"
-            duration="Ongoing weekly rewards"
-            text="Provide any amount of XUSD to the XUSD lending pool and instantly start accruing your share of 15,000 SOV rewards."
+            title={t(translations.promotions.card2.title)}
+            duration={t(translations.promotions.card2.duration)}
+            text={t(translations.promotions.card2.text)}
             linkAsset={Asset.XUSD}
-            learnMoreLink="https://www.sovryn.app/blog/sov-is-diving-into-lending-pools"
+            learnMoreLink="https://wiki.sovryn.app/en/sovryn-dapp/market-making"
```
Comment (tiltom) [inline src/app/components/Promotions/index.tsx:40]: I would extract the link to `utils/classifiers`.

File: src/utils/dictionaries/amm/mainnet.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export const mainnetAmm = [
     '0xe76Ea314b32fCf641C6c57f14110c5Baa1e45ff4',
     '0x09C5faF7723b13434ABdF1a65aB1B667BC02A902',
   )
-    .setLootDropColor(LootDropColors.Purple)
+    .setLootDropColor(PromotionColor.Purple)
```
Comment (tiltom) [inline src/utils/dictionaries/amm/mainnet.ts:15]: Rename the method to `setPromotionColor`.

File: src/utils/models/amm-liquidity-pool.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ import LiquidityPoolV2Converter from '../blockchain/abi/LiquidityPoolV2Converter
 export type ConverterVersion = 1 | 2;
 
 export class AmmLiquidityPool {
-  private _lootDropColor?: LootDropColors | string;
+  private _lootDropColor?: PromotionColor | string;
```
Comment (tiltom) [inline src/utils/models/amm-liquidity-pool.ts:10]:
```suggestion
  private _promotionColor?: PromotionColor | string;
```

File: src/utils/models/amm-liquidity-pool.ts [hunk 5]
```diff
@@ -35,7 +35,7 @@ export class AmmLiquidityPool {
       this.poolTokenB = poolTokenB.toLowerCase();
     }
   }
-  public setLootDropColor(color: LootDropColors | string) {
+  public setLootDropColor(color: PromotionColor | string) {
```
Comment (tiltom) [inline src/utils/models/amm-liquidity-pool.ts:38]:
```suggestion
  public setPromotionColor(color: PromotionColor | string) {
```

**Review by tiltom — CHANGES_REQUESTED** [review]

**Comment by soulBit** [comment]
Other than tiltom's comments, changes and code look good generally - added some suggestions to Monday story which should be reviewed/implemented as appropriate

File: src/app/components/Promotions/index.tsx (hunk 1 above)
Comment (pietro-maximoff) [reply src/app/components/Promotions/index.tsx:28]: done

File: src/utils/dictionaries/amm/mainnet.ts (hunk 3 above)
Comment (pietro-maximoff) [reply src/utils/dictionaries/amm/mainnet.ts:15]: done

File: src/utils/models/amm-liquidity-pool.ts (hunk 5 above)
Comment (pietro-maximoff) [reply src/utils/models/amm-liquidity-pool.ts:38]: done

**Comment by pietro-maximoff** [comment]
looks good to me

**Review by soulBit — APPROVED** [review]

**Pull request merged**